assets/src/edit-story/components/form/numeric.js

```diff
@@ -79,6 +79,13 @@ function Numeric({
   const placeholder = isMultiple ? __('multiple', 'web-stories') : '';
   const [focused, setFocus] = useState(false);
   const [dot, setDot] = useState(false);
+  const ref = useRef();
+
+  useEffect(() => {
+    if (focused && ref.current) {
```

Comment:
So, mouse focusing also does select-all?

Reply:
Yes, as mentioned in PR description. It's "by design" 😉.